src/Umbraco.Forms.Integrations.Crm.Hubspot/App_Plugins/UmbracoForms.Integrations/Crm/Hubspot/hubspot-field-mapper-template.html

```diff
@@ -24,6 +24,7 @@
             <div class="umb-forms-mapping-header">
                 <div class="umb-forms-mapping-field -no-margin-left">Form Field</div>
                 <div class="umb-forms-mapping-field">Hubspot Field</div>
+                <div class="umb-forms-mapping-field text-center">Append when updating values</div>
```

Comment:
I think this heading and the checkbox should be hidden if `AllowContactUpdate` is set to false.

To do that, you'll need a way of getting values from configuration into the client-side so you can act on that.  There's a technique for doing that where we add values to the "server variables".

For example in Forms this is done [here](https://github.com/umbraco/Forms/blob/v10/dev/src/Umbraco.Forms.Core/NotificationHandlers/ServerVariablesParsingHandler.cs#L69).

Which then makes it available [here](https://github.com/umbraco/Forms/blob/v10/dev/src/Umbraco.Forms.StaticAssets/app/backoffice/Form/overlays/fieldsetsettings/fieldsetsettings.controller.js#L83) client-side.

Reply:
Updated